scripts/lb-wrapper

```diff
@@ -1,48 +1,50 @@
 #!/bin/bash
 
 LOG_FILE="/var/log/xklb.log"
+LOG_LEVEL="-vv"
 XKLB_EXECUTABLE="${XKLB_EXECUTABLE:-lb}"
 TMP_DOWNLOADS_DIR="/library/downloads/calibre-web"
 SURVEY_DB_FILE="${TMP_DOWNLOADS_DIR}/survey.db"
 URL="$1"
 FORMAT_OPTIONS="--format best --format-sort 'tbr~1000'"
```

Comment:
Please delete scripts/lb-wrapper.greedy if you commit this suggestion: (as we discussed a few days ago)

```suggestion

# Or download largest possible HD-style / UltraHD videos, to try to force
# out-of-memory "502 Bad Gateway" for testing of issues like #37 and #79
# FORMAT_OPTIONS="--format-sort size"

FORMAT_OPTIONS="--format best --format-sort 'tbr~1000'"
```

Reply:
> Please delete scripts/lb-wrapper.greedy if you commit this suggestion: (as we discussed a few days ago)

@deldesir see above request.

(This thread appears to have been was closed prematurely.)

Reply:
I get it. I will delete lb-wrapper.greedy